lib/tasks/backfill_in_person_pending_at.rake

```diff
@@ -0,0 +1,68 @@
+namespace :profiles do
+  desc 'Backfill the in_person_verification_pending_at value column.'
+
+  ##
+  # Usage:
+  #
+  # Print pending updates
+  # bundle exec rake profiles:backfill_in_person_verification_pending_at
+  #
+  # Commit updates
+  # bundle exec rake profiles:backfill_in_person_verification_pending_at UPDATE_PROFILES=true
+  #
+  task backfill_in_person_verification_pending_at: :environment do |_task, _args|
+    ActiveRecord::Base.connection.execute('SET statement_timeout = 60000')
+
+    update_profiles = ENV['UPDATE_PROFILES'] == 'true'
+
+    profiles = Profile.where(deactivation_reason: 'in_person_verification_pending')
+
+    profiles.each do |profile|
+      timestamp = profile.in_person_enrollment.updated_at
+
+      warn "#{profile.id},#{profile.deactivation_reason},#{timestamp}"
+      profile.update!(in_person_verification_pending_at: timestamp) if update_profiles
```

Comment:
I think we want to set deactivation reason to nil in the same transaction as we set the timestamp.

Reply:
Changed